### PR TITLE
Fix code style for hosting_two_column template

### PR DESCRIPTION
**Pull request by KorvinSzanto**
The PR normalizes the code style to match our code style requirements. Most of the work here was done by running "reformat code" in PHPStorm but it needed a couple tweaks by hand to be correct.

### CODE REVIEW

**Comment by KorvinSzanto** [comment]
https://github.com/concretecms/concrete_cms_theme/pull/77/files?w=1 is a little easier to read

File: blocks/express_form/templates/hosting_two_column.php [hunk 1]
```diff
@@ -43,23 +50,27 @@
...
+                            <?php
+                        }
+                        ?>
 
                         <div><?php $captcha->display(); ?></div>
                         <div><?php $captcha->showInput(); ?></div>
                     </div>
-                <?php } ?>
+                <?php
+                } ?>
```
Comment (lrcg) [inline blocks/express_form/templates/hosting_two_column.php:62]: Closing tag inconsistent with lines:
* 27-29
* 37-39
* 54-56
* 73-75

File: blocks/express_form/templates/hosting_two_column.php [hunk 2]
```diff
@@ -12,27 +12,34 @@
...
-        ?>
+            <?php
+            if (isset($error) && is_object($error)) {
+                ?>
                 <div class="alert alert-danger">
-                    <?=$error->output(); ?>
+                    <?= $error->output(); ?>
                 </div>
-            <?php } ?>
+            <?php
```
Comment (lrcg) [inline blocks/express_form/templates/hosting_two_column.php:37]: Leading whitespace inconsistent with lines:
* 27
* 54
* 69
* 73

File: blocks/express_form/templates/hosting_two_column.php [hunk 3]
```diff
@@ -43,23 +50,27 @@
...
+                            <label class="control-label form-label"><?= $captchaLabel ?></label>
+                            <?php
+                        }
+                        ?>
 
                         <div><?php $captcha->display(); ?></div>
                         <div><?php $captcha->showInput(); ?></div>
                     </div>
-                <?php } ?>
+                <?php
```
Comment (lrcg) [inline blocks/express_form/templates/hosting_two_column.php:61]: Leading whitespace inconsistent with lines:
* 27
* 54
* 69
* 73

**Review by lrcg — COMMENTED** [review]
Inconsistency in leading whitespace and newlines

**Review by wordish — APPROVED** [review]

**Pull request merged**